### PR TITLE
Cleanup docstrings for `gammapy.makers`

### PR DESCRIPTION
**Pull request by Astro-Kirsty**
This addresses part of issue https://github.com/gammapy/gammapy/issues/4811, specifically for `gammapy.makers`

### CODE REVIEW

File: gammapy/makers/background/phase.py [hunk 1]
```diff
@@ -16,11 +16,12 @@ class PhaseBackgroundMaker(Maker):
     Parameters
     ----------
     on_phase : `tuple` or list of tuples
-        on-phase defined by the two edges of each interval (edges are excluded).
+        Pn-phase defined by the two edges of each interval (edges are excluded).
```
Comment (registerrier) [inline gammapy/makers/background/phase.py:19]:
```suggestion
        On-phase defined by the two edges of each interval (edges are excluded).
```

File: gammapy/makers/background/phase.py [hunk 2]
```diff
@@ -16,11 +16,12 @@ class PhaseBackgroundMaker(Maker):
...
     off_phase : `tuple` or list of tuples
-        off-phase defined by the two edges of each interval (edges are excluded).
-    phase_column_name : `str`
-        The name of the column in the event file from which the phase informations are extracted. Default is 'PHASE'.
+        Pff-phase defined by the two edges of each interval (edges are excluded).
```
Comment (registerrier) [inline gammapy/makers/background/phase.py:21]:
```suggestion
        Off-phase defined by the two edges of each interval (edges are excluded).
```

**Review by registerrier — APPROVED** [review]
Thanks @Astro-Kirsty . Looks good.

File: gammapy/makers/background/reflected.py [hunk 3]
```diff
@@ -80,7 +77,7 @@ def is_rad_max_compatible_region_geom(rad_max, geom, rtol=1e-3):
 
 
 class RegionsFinder(metaclass=ABCMeta):
-    """Baseclass for regions finders
+    """Baseclass for regions finders.
```
Comment (Astro-Kirsty) [inline gammapy/makers/background/reflected.py:80]:
```suggestion
    """Base class for regions finders.
```

File: gammapy/makers/background/reflected.py [hunk 4]
```diff
@@ -145,17 +143,17 @@ def _create_reference_geometry(self, region, center):
 
     @staticmethod
     def _get_center_pixel(center, reference_geom):
-        """Center pix coordinate"""
+        """Center pix coordinate."""
```
Comment (Astro-Kirsty) [inline gammapy/makers/background/reflected.py:146]:
```suggestion
        """Center pixel coordinate."""
```

**Review by registerrier — APPROVED** [review]
Thanks @Astro-Kirsty

**Pull request merged**